docs/source/slack.rst

```diff
@@ -63,7 +63,7 @@ Instead you can either share code as media on Slack (see screenshot below), or
 upload your code to paste service like paste.debian.net or gist.github.com and
 then share a link with us.
 
-.. image:: img/slack-code.png
+.. image:: img/slack-dark-1.png
```

Comment:
Ideally, we would want to get rid of the old file (`img/slack-code.png`).

One of the ways of doing so is deleting the old file, and renaming `img/slack-dark-1.png` to `img/slack-code.png` (this way, we won't need to change any documentation at all).